src/tls/lib.rs

```diff
@@ -68,14 +68,24 @@ pub(super) fn provider() -> Arc<CryptoProvider> {
 
 #[cfg(feature = "tls-aws-lc")]
 pub(super) fn provider() -> Arc<CryptoProvider> {
-    Arc::new(CryptoProvider {
+    let mut provider = CryptoProvider {
         // Limit to only the subset of ciphers that are FIPS compatible
         cipher_suites: vec![
             rustls::crypto::aws_lc_rs::cipher_suite::TLS13_AES_256_GCM_SHA384,
             rustls::crypto::aws_lc_rs::cipher_suite::TLS13_AES_128_GCM_SHA256,
         ],
         ..rustls::crypto::aws_lc_rs::default_provider()
-    })
+    };
+
+    let kx_groups = match std::env::var("COMPLIANCE_POLICY").as_deref() {
```

Comment:
In every other part of the codebase we do not read raw env vars and instead pass them into config::Config.

For this function Ican see how passing the config around may be tedious. So if we are going to do it can we at least read it once and store it in a global var (once_cell::sync::Lazy can be used)?

Reply:
Done